src/potentials/rebo/nonreactive.rs

```diff
@@ -159,10 +159,10 @@ pub const SITE_MAX_BONDS: usize = 4;
 #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, enum_map::Enum)]
 pub enum AtomType { Carbon, Hydrogen }
 impl AtomType {
-    pub fn char(self) -> char {
+    pub fn char(self) -> &'static str {
```

Comment:
Oops!  `char` definitely was a mistake here in hindsight, though I think I'd rather have this changed to `pub fn element(self) -> Element`.  (and code that used to call `.char()` can call `.element().symbol()` instead)